modules/swagger-codegen/src/main/resources/typescript-angular2/encoder.mustache

```diff
@@ -0,0 +1,17 @@
+/*
+* CustomQueryEncoderHelper
+* Fix plus sign (+) not encoding, so sent as blank space
+* See: https://github.com/angular/angular/issues/11058#issuecomment-247367318
+*/
```

Comment:
You could think about moving this to line 7, and making it a docstring `/**` so we *could* use some tooling like Typedoc in the future. Just a very minor suggestion though :)

Reply:
Right! I'll fix it soon.